### ISSUE TITLE
Data from mapbox

### ISSUE THREAD

**Issue by naruarjun** [jackkwok/neural-road-inspector#2]
Hey! I wanted to know how to create the GeoJSON file to create the dataset in mapbox. Any help regarding how to get data from mapbox will be greatly appreciated

**Comment by Geoyi** [comment]
Sorry to hijack the question here @jackkwok. 
@naruarjun, you can look into use [Label Maker](https://github.com/developmentseed/label-maker) to create full training dataset that will pair Mapbox's satellite imagery and the OSM roads layer for you, and I have [a notebook](https://github.com/Geoyi/FOSS_4g_Pixel_decoder/blob/master/Label-Maker_Pixel-Decoder.ipynb) to explain how to use it in another [road detection neural network package, Pixel Decoder](https://github.com/Geoyi/pixel-decoder) I worked on too.
